core/src/test/java/org/elasticsearch/index/query/TemplateQueryBuilderTests.java

```diff
@@ -80,7 +80,8 @@ protected void assertBoost(TemplateQueryBuilder queryBuilder, Query query) throw
     @Override
     public void testUnknownField() throws IOException {
         TemplateQueryBuilder testQuery = createTestQueryBuilder();
-        String queryAsString = testQuery.toString().replace("inline", "bogusField");
+        String testQueryAsString = toXContent(testQuery, randomFrom(XContentType.JSON, XContentType.YAML)).string();
```

Comment:
Just checking AbstractQueryBuilder#toString() which is implemented in ToXContentToBytes. Seems to always use json builder, so I'm not sure if for this kind of test (and the ones mentioned) we need aditional Yaml variant, but its okay to add it I guess.